Prevent oops on module removal when we don't have the device available.

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com>

---
 linux-2.6.23/3rdparty/aes2501/aes2501.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff -p -up linux-2.6.23/3rdparty/aes2501/aes2501.c.orig linux-2.6.23/3rdparty/aes2501/aes2501.c
--- linux-2.6.23/3rdparty/aes2501/aes2501.c.orig	2008-01-13 23:37:38.000000000 -0200
+++ linux-2.6.23/3rdparty/aes2501/aes2501.c	2008-01-13 23:38:26.000000000 -0200
@@ -1616,7 +1616,8 @@ static int __init aes2501_init(void)
 
 static void __exit aes2501_exit(void)
 {
-	_dev->stop_scan = 1;
+	if (_dev)
+		_dev->stop_scan = 1;
 	destroy_workqueue(comm_queue);
 	usb_deregister(&aes2501_driver);
 }
